src/hotspot/cpu/x86/stubGenerator_x86_64_fmod.cpp

```diff
@@ -292,11 +292,6 @@ address StubGenerator::generate_libmFmod() {
     //   __asm { stmxcsr DWORD PTR[mxcsr] }
     //   mxcsr_rz = 0x7f80 | mxcsr;
     __ push(rax);
-    __ stmxcsr(Address(rsp, 0));
-    __ movl(rax, Address(rsp, 0));
-    __ movl(rcx, rax);
-    __ orl(rcx, 0x7f80);
-    __ movl(Address(rsp, 0x04), rcx);
```

Comment:
No need to do stmxcsr, we could define mxcsr_rz on similar lines as mxcsr_std as part of StubGenerator::create_control_words() and use those in fmod.